src/types.ts

```diff
@@ -1,8 +1,16 @@
-import { PieceType, Square } from '../cjsmin/src/chess';
+import { Square, UASymbol } from '../cjsmin/src/chess';
 
 export interface FileReaderGame {
```

Comment:
assuming that FileReaderGame still used by GameChunks

Reply:
and other things now